winui/AutoComplete/No-Result-Found.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: No Result Found in WinUI AutoComplete control | Syncfusion
+description: Learn how to use NoResultsFoundTemplate and NoResultsFoundContent in Syncfusion WinUI AutoComplete control.
+platform: winui
+control: SfAutoComplete
+documentation: ug
+---
+
+# No Result Found  in WinUI AutoComplete (SfAutoComplete)

```

Comment:
@YuvarajMasilamani, remove extra space in between found and in

Reply:
resolved